draft-ietf-httpapi-privacy.md

```diff
@@ -136,19 +136,29 @@ authentication SHOULD include the Secure attribute described in {{Section
 4.1.2.5 of RFC6265}}. Bearer tokens MAY use the format described in {{?RFC8959}}
 to indicate the expected usage to the client.
 
-## Credential Revocation
+## Disclosure Response
 
 Some deployments may not find it feasible to completely block unencrypted
 connections, whether because the hostname is shared with unauthenticated
 endpoints or for infrastructure reasons. Therefore, servers need a response for
 when a valid credential has been received over an insecure channel.
```

Comment:
```suggestion
when a credential has been received over an insecure channel.
```

The paragraphs below mention that the response should be the same regardless of whether the credentials are valid or not. So I think we should not explicitly mention "valid credentials" here.